src/models/generated/MarketBidCost.jl

```diff
@@ -0,0 +1,92 @@
+#=
+This file is auto-generated. Do not edit.
+=#
+"""
+    mutable struct MarketBidCost <: OperationalCost
+        energy_bid::VariableCost
+        no_load_cost::Float64
+        start_up::NamedTuple{(:hot, :int, :cold), NTuple{3, Float64}}
```

Comment:
`int`? I'd prefer `warm` if it's appropriate, or `intermediate` ...

Reply:
+1 for changing this name, `warm` seems better

Reply:
I had warm but the forms ask for intermediate as the name of the field. But if this is preferred I will fix it.